app/routing/ArrivalsRouter.scala

```diff
@@ -42,4 +44,10 @@ class ArrivalsRouter @Inject() (
       case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE) => v2Arrivals.createArrivalNotification()
```

Comment:
VERSION_2_ACCEPT_HEADER_VALUE looks outdated, should be VERSION_2_ACCEPT_HEADER_VALUE_JSON - need to merge with main